nutkit/protocol/requests.py

```diff
@@ -87,6 +88,7 @@ def __init__(
         self.livenessCheckTimeoutMs = liveness_check_timeout_ms
         self.maxConnectionPoolSize = max_connection_pool_size
         self.connectionAcquisitionTimeoutMs = connection_acquisition_timeout_ms
+        self.notificationFilters = notification_filters
```

Comment:
Why is `None` a serialized value here, but not in `NewSession`? I think it should at least be consistent. I'm unsure which option is better though. My initial gut feeling tells me to always put it on the wire.

Reply:
good spot, I messed up, I'll serialize `None` on both.